pioneer/packages/joy-media/src/Upload.tsx

```diff
@@ -248,8 +248,8 @@ class Component extends React.PureComponent<Props, State> {
 
     // TODO get corresponding data type id based on file content
     const dataObjectTypeId = new BN(1);
-
-    return [newContentId, dataObjectTypeId, new BN(file.size), ipfs_cid];
+    const { myMemberId } = this.props;
+    return [newContentId, myMemberId, dataObjectTypeId, new BN(file.size), ipfs_cid];
```

Comment:
Actually, now I noticed the order of the parameters is incorrect too.
It should be: `return [myMemberId, newContentId (...)`

Reply:
fixed in d5641f18260248352719db6b980fcc3dbcc44ede